data/colors_v2/vars/global_dark.ts

```diff
@@ -21,7 +21,7 @@ export default {
   shadow: {
     small: '0 0 transparent',
     medium: (theme: any) => `0 3px 6px ${get('scale.black')(theme)}`,
-    large: (theme: any) => `0 8px 24px ${get('scale.black')(theme)}`,
+    large: (theme: any) => `0px 0px 0px 1px ${get('scale.gray.6')(theme)}, 0px 16px 32px ${alpha(get('scale.black'), 0.85)(theme)}, 0px 1px 3px ${alpha(get('scale.black'), 0.12)(theme)}`,
```

Comment:
This 1px gray border might conflict with modals that already have a border, right?

Reply:
I think so, this should be one of the things to test? It became a little difficult to test these in Figma. I'll let @vdepizzol chime in as well.

Reply:
Tested this a bit.. the output is:

```css
--color-shadow-large: 0px 0px 0px 1px #30363d, 0px 16px 32px rgba(1,4,9,0.85), 0px 1px 3px rgba(1,4,9,0.12);
```

... and yes.. it could add a "double border" wherever a "normal" border already exists. For example the select menus already have a border and would look like:

<img width="370" alt="Screen Shot 2021-08-10 at 19 53 17" src="https://user-images.githubusercontent.com/378023/128855134-ccaad0e2-9b50-408b-b415-aec231b2697d.png">

I actually kinda like it.. 😆  but yeah, we would need to make sure we use the 2px border on other things too.

If we wanna keep it to 1px we would need to make sure to refactor all places that use `--color-shadow-large` and make sure it doesn't already have a border. Hmmm.. maybe out of scope for GA?

Reply:
Tested this a bit.. the output is:

```css
--color-shadow-large: 0px 0px 0px 1px #30363d, 0px 16px 32px rgba(1,4,9,0.85), 0px 1px 3px rgba(1,4,9,0.12);
```

... and yes.. it could add a "double border" wherever a "normal" border already exists. For example the select menus already have a border and would look like:

<img width="370" alt="Screen Shot 2021-08-10 at 19 53 17" src="https://user-images.githubusercontent.com/378023/128855134-ccaad0e2-9b50-408b-b415-aec231b2697d.png">

I actually kinda like it.. 😆  but yeah, we would need to make sure we use the 2px border on other things too.

If we wanna keep it to 1px we would need to make sure to refactor all places that use `--color-shadow-large` and make sure it doesn't already have a border. Hmmm.. maybe out of scope for GA?